pkg/apis/core/validation/utils_test.go

```diff
@@ -139,4 +140,56 @@ var _ = Describe("Utils tests", func() {
 			Expect(errorList).To(BeEmpty())
 		})
 	})
+
+	Describe("#Validate", func() {
+		var seed *core.Seed
+
+		BeforeEach(func() {
+			seed = &core.Seed{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-seed",
+				},
+				Spec: core.SeedSpec{
+					Provider: core.SeedProvider{
+						Type:   "test-provider",
+						Region: "test-region",
+					},
+					Ingress: &core.Ingress{
+						Domain: "someingress.example.com",
+						Controller: core.IngressController{
+							Kind: "nginx",
+						},
+					},
+					DNS: core.SeedDNS{
+						Provider: &core.SeedDNSProvider{
+							Type: "provider",
+							SecretRef: corev1.SecretReference{
+								Name:      "secret",
+								Namespace: "garden",
+							},
+						},
+					},
+					Networks: core.SeedNetworks{
+						Pods:     "10.123.211.10/18",
+						Services: "193.168.211.0/16",
+					},
+					Settings: &core.SeedSettings{
+						DependencyWatchdog: &core.SeedSettingDependencyWatchdog{},
+					},
+				},
+			}
+		})
+
+		It("should allow if deprecated fields are not set", func() {
+			seed.Spec.Settings.DependencyWatchdog.Weeder = &core.SeedSettingDependencyWatchdogWeeder{
+				Enabled: true,
+			}
+			seed.Spec.Settings.DependencyWatchdog.Prober = &core.SeedSettingDependencyWatchdogProber{
+				Enabled: true,
+			}
+			allErrs := ValidateSeed(seed)
+
+			Expect(allErrs).To(BeEmpty())
+		})
+	})
```

Comment:
Same question here. The validation code seems not to be adapted, so what's the purpose of all this?

Reply:
earlier I had added validation here thats why, I have removed the test case